packages/astro/components/Code.astro

```diff
@@ -37,7 +37,7 @@ const { code, lang = 'plaintext', theme = 'github-dark', wrap = false } = Astro.
 /** Replace the shiki class name with a custom astro class name. */
 function repairShikiTheme(html: string): string {
 	// Replace "shiki" class naming with "astro"
-	html = html.replace('<pre class="shiki"', '<pre class="astro-code"');
+	html = html.replace(/<pre class="(.*)shiki(.*)"/, '<pre class="$1astro-code$2"');
```

Comment:
```suggestion
	html = html.replace(/<pre class="(.*?)shiki(.*?)"/, '<pre class="$1astro-code$2"');
```

I think it would be nice to make this a bit stricter so it doesn't match `<pre class="shiki boom"><code class="bam">` incorrectly. If shiki ever decide to do that one day